src/components/MessagingView.tsx

```diff
@@ -4,333 +4,943 @@ import {FhirClientContext, FhirClientContextType} from "../FhirClientContext";
+              activeMessage: { ...this.state.activeMessage, type: value },
+            });
+          }}
+          textColor="primary"
+          indicatorColor="primary"
+          aria-label="secondary tabs example"
+          variant="scrollable"
+          scrollButtons="auto"
+          allowScrollButtonsMobile
+          sx={{
+            marginTop: 1.5,
+            marginBottom: 1,
+            padding: (theme) => theme.spacing(0, 1, 0),
+            borderBottom: 1,
+            borderColor: "divider",
+            minHeight: "40px"
+          }}
+        >
+          <Tab value="sms" label="Issac send" {...tabProps}/>
+          <Tab value="manual" label="Enter manual message"  {...tabProps}/>
+          <Tab value="comment" label="Enter comment" {...tabProps} />
+        </Tabs>
+        <IconButton
+          color="info"
+          size="small"
+          onClick={() => this.setState({ infoOpen: true })}
+        >
+          <InfoIcon></InfoIcon>
+        </IconButton>
+        <Dialog
+          open={this.state.infoOpen}
+          onClose={handleInfoClose}
+          aria-labelledby="non-sms-dialog"
+          aria-describedby="non-sms-dialog-description"
+        >
+          <DialogTitle>Information about message types</DialogTitle>
+          <DialogContent>
+            <Typography variant="body1">
+              <p>
+                <b>ISACC send</b> are messages sent or received directly by
+                ISACC, which include texts, “sms”, iMessages, etc.
+              </p>
+              <p>
+                <b>Manual messages</b> mean messages external to ISACC,
+                including manually entered notifications of emails, phone calls,
+                postal mail, or other non-automated communication.
+              </p>
+              <p>
+                <b>Comments</b> are contact/note on the recipient.
+              </p>
             </Typography>
-        </Box>;
-
-        let bubbleAndPriorityRow;
-        if (incoming) { // different order based on message direction
-            bubbleAndPriorityRow = <>
-                {box}
-                {priorityIndicator}
-            </>
-        } else {
-            bubbleAndPriorityRow = <>
-                {priorityIndicator}
-                {box}
-            </>
+            <Alert severity="info">
+              Please note that messages external to ISACC and contact/note on
+              the recipient will not be sent/communicated to the recipient.
+            </Alert>
+          </DialogContent>
+          <DialogActions>
+            <Button variant="text" onClick={handleInfoClose}>
+              Close
+            </Button>
+          </DialogActions>
+        </Dialog>
+      </Stack>
+    );
+  }
+
+  private _buildSMSEntryComponent(): React.ReactNode {
+    return (
+      <>
+        <TextField
+          multiline
+          value={this.state.activeMessage?.content ?? ""}
+          placeholder={"Type message for ISSAC to send"}
```

Comment:
Two Cs, as in *C*aring *C*ontacts :)
```suggestion
          placeholder={"Type message for ISACC to send"}
```